index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <div class="main-container">
+    <header class="logo">
+      <h1 class="me">Habtamu</h1>
+      <nav class="humberger">
+        <a href="#"><img class="vec" src="images/Union.png" alt=""></a>
+      </nav>
+  </header>
+  <section class="initial">
+    <h1 class="information">Hey There.
+    I’m Habtamu </h1>
+    <h2>I’am a Software Developer</h2>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.
+    </p>
```

Comment:
It's almost done, but please try to use the same font, font-size, font-weight, and line-height as it is in the Figma template. This will make your design look much better.

![image](https://user-images.githubusercontent.com/34779928/198702397-6b5dd890-4767-4a49-9af8-3f88100c4e7e.png)